net-irc/znc/files/znc-1.6.3-libressl.patch

```diff
@@ -0,0 +1,19 @@
+$OpenBSD: patch-src_Csocket_cpp,v 1.1 2014/07/12 14:42:37 pascal Exp $
+--- a/src/Csocket.cpp.orig	Sat Jul 12 16:03:48 2014
++++ b/src/Csocket.cpp	Sat Jul 12 16:04:36 2014
```

Comment:
1.6.3 should work with libressl without this patch, because it now has the right `#include`s